### PR TITLE
[TEVA-2687] Send email and destroy inactive jobseeker accounts

### PR DESCRIPTION
**Pull request by cesidio**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-2687

## Screenshots of UI changes:
Screenshot 2021-07-20 at 15 27 46

### CODE REVIEW

File: config/locales/mailers.yml [hunk 1]
```diff
@@ -47,6 +47,12 @@ en:
           body: If you did not make this request, please email %{mail_to}
           heading: Not requested this?
         subject: Email updated
+      inactive_account:
+        intro: We will delete your Teaching Vacancies account and data unless you sign into your account in the next 2 weeks.
+        explain: This is because you have not used your account for 5 years.
```
Comment (josephhull676) [inline config/locales/mailers.yml:52]: Tiny thing but could 'explanation' be better?

File: app/jobs/send_inactive_account_email_job.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class SendInactiveAccountEmailJob < ApplicationJob
+  queue_as :default
+
+  def perform_now
+    return if DisableExpensiveJobs.enabled?
+
+    Jobseeker.where("DATE(last_sign_in_at) = ?", 5.years.ago.to_date).each do |jobseeker|
```
Comment (josephhull676) [inline app/jobs/send_inactive_account_email_job.rb:7]: So this sends the email to jobseekers who last signed in 5 years ago? Shouldn't it be jobseekers who last signed in 2 weeks less than 5 years ago?

File: app/jobs/destroy_inactive_accounts_job.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class DestroyInactiveAccountsJob < ApplicationJob
+  queue_as :default
+
+  def perform_now
+    return if DisableExpensiveJobs.enabled?
+
+    Jobseeker.where("DATE(last_sign_in_at) = ?", (5.years.ago - 2.weeks).to_date).each do |jobseeker|
```
Comment (josephhull676) [inline app/jobs/destroy_inactive_accounts_job.rb:7]: Where as this should be 5 years ago?

**Review by josephhull676 — COMMENTED** [review]
Left some comments!

**Comment by josephhull676** [comment]
Ignore the comments. Just re-read the ticket and realised the email is sent after 5 years and the account is destroyed two weeks later. Also realised `5.years.ago - 2.weeks` is 5 years and 2 weeks ago.

File: config/schedule.yml [hunk 4]
```diff
@@ -3,6 +3,11 @@ clear_emergency_login_keys:
   class: 'ClearEmergencyLoginKeysJob'
   queue: low
 
+destroy_inactive_accounts:
+  cron: '10 8 * * *'
```
Comment (alexbowen) [inline config/schedule.yml:7]: im no an expert with cron syntax but its ok that this has 8 when other below has 09 (instead of 9)?

**Review by alexbowen — APPROVED** [review]
minor query else lgtm

File: config/schedule.yml (hunk 4 above)
Comment (cesidio) [reply config/schedule.yml:7]: some do some don't. 

https://crontab.guru/

I agree we should standardise on one approach, but it's a very minor thing

**Pull request merged**